maat-court-data-api/src/test/java/gov/uk/courtdata/applicant/controller/ApplicantControllerTest.java

```diff
@@ -56,16 +57,63 @@ void givenAValidationException_whenGetRepOrderApplicantLinksIsInvoked_thenCorrec
 
     @Test
     void givenCorrectRepId_whenGetRepOrderApplicantLinksIsInvoked_thenResponseIsReturned() throws Exception {
-        List<RepOrderApplicantLinksDTO> response = TestModelDataBuilder.getRepOrderApplicantLinksDTO();
+        List<RepOrderApplicantLinksDTO> response = TestModelDataBuilder.getRepOrderApplicantLinksDTOs(ID);
         when(repOrderApplicantLinksService.find(REP_ID)).thenReturn(response);
         mvc.perform(MockMvcRequestBuilders.get(ENDPOINT_URL + "/rep-order-applicant-links/" + REP_ID))
                 .andExpect(status().isOk());
         verify(repOrderApplicantLinksService).find(REP_ID);
```

Comment:
If we've mocked the call with `when`, then there's no need for a `verify` call here.